gprotorch/dataloader/mol_prop.py

```diff
@@ -73,7 +79,12 @@ def featurize(self, representation, bond_radius=3, nBits=2048):
         def fingerprints():
 
             rdkit_mols = [MolFromSmiles(smiles) for smiles in self.features]
-            fps = [AllChem.GetMorganFingerprintAsBitVect(mol, bond_radius, nBits=nBits) for mol in rdkit_mols]
+            fps = [
```

Comment:
This is Black's suggestion with the formatting I'm guessing?

Reply:
yh